### PR TITLE
remove python3-nose

### PR DESCRIPTION
**Pull request by SJLC**
nose test framework is highly deprecated, having been unmaintained for a
long time. There is just one dependency on nose in this layer,
python3-flake8-logging-format, which turns out to be a spurious
dependency that can be fixed by a simple patch.

python3-flake8-logging-format incorrectly listed nose as a
standard requirement, rather than listing it with the test dependencies,
but nose is clearly a test requirement *only* used when running the
unit tests. No functionality of the package itself depends on nose,
so moving it to test_require is the right thing to do for now
(i.e. until someone updates the unit tests to run under a currently
supported test framework).

Signed-off-by: S. Lockwood-Childs <sjl@vctlabs.com>

### CODE REVIEW

File: recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+From f1bf907cc14f55edebac9d4633a4c1edec0f9f53 Mon Sep 17 00:00:00 2001
+From: S. Lockwood-Childs <sjl@vctlabs.com>
+Date: Fri, 4 Mar 2022 12:53:16 -0800
+Subject: [PATCH] Remove hard dependency on nose
+
+nose is not really a runtime dependency, but rather just a dependency
+for running the unit tests.
+
+Signed-off-by: S. Lockwood-Childs <sjl@vctlabs.com>
+---
```
Comment (priv-kweihmann) [inline recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch:10]: Could you please propose this change upstream first - I would rather like to remove the nose recipe as part of an update of this package

Comment (priv-kweihmann) [reply recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch:10]: And if upstream is dead, I'd like to see the corresponding `Upstream-Status` meta data entry here

Comment (SJLC) [reply recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch:10]: Ok I can try

Comment (SJLC) [reply recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch:10]: https://github.com/globality-corp/flake8-logging-format/pull/30

Comment (priv-kweihmann) [reply recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch:10]: Thx

**Pull request merged**